api/src/main/java/org/apache/iceberg/view/ViewRepresentation.java

```diff
@@ -18,21 +18,16 @@
  */
 package org.apache.iceberg.view;
 
-import java.util.Locale;
+import org.immutables.value.Value;
```

Comment:
Same here: `javax.annotation.Nullable`

Reply:
ViewRepresentations currently has a single required "type" field so it wasn't marked as Nullable to begin with. Let me know if I'm missing something!